client/src/modules/archived/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Toolbar from 'common/components/Toolbar';
+import { fetchArchivedListsMetaData } from 'modules/shopping-list/model/actions';
+import { getShoppingLists } from 'modules/shopping-list/model/selectors';
+import { ListIcon } from 'assets/images/icons';
+import GridList from 'common/components/GridList';
+
+class Archived extends PureComponent {
+  componentDidMount() {
+    const { fetchArchivedListsMetaData } = this.props;
+
+    fetchArchivedListsMetaData();
+  }
+
+  render() {
+    const { lists } = this.props;
+
+    return (
+      <Fragment>
+        <Toolbar />
+        <div className="wrapper">
+          <GridList
+            icon={<ListIcon />}
+            name="Archived Lists"
+            items={lists}
+            message="There are no archived lists!"
```

Comment:
consider renaming to placeholder or something simmiliar, cause this props should clearly say thata this text will be displayed when grid list is empty. 

placeholderMessage or just placeholder